devicetypes/rooms-beautiful/rooms-beautiful-curtain.src/rooms-beautiful-curtain.groovy

```diff
@@ -21,16 +21,18 @@ metadata {
 		capability "Battery"
 		capability "Configuration"
 		capability "Refresh"
-		capability "Window Shade"
-		capability "Health Check"
+        capability "Health Check"
+		capability "Window Shade"		
+        capability "Switch"
+        capability "Switch Level"
```

Comment:
Just so you know, we're in the process of adding a new `Window Shade Level` capability which will be preferred over `Switch Level` for devices like this. 

@dkirker do you think we're ready to use that new capability right now or should they stick with `Switch Level` for the moment?

Reply:
I would still like to use Switch Level as it is compatible with Alexa.

Reply:
I have filed a request to make the Alexa integration work with `Window Shade` and `Window Shade Level` but this is fine for now.

Reply:
@tpmanley Window Shade Level isn't quite ready. I think @larsfinander did add it, at least Window Shade, to the voice integration stuff. I'll put that on the tracker I have been using for Window Shades.